assignments/objects.js

```diff
@@ -49,16 +85,37 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  'name': 'Susan',
+  'age': 70,
+  sayName: function() {
+    return this['name'];
+  },
+  'child': {
+    'name': 'George',
+    'age': 50,
+    sayName: function() {
+      return this['name'];
+    },
+    'child': {
```

Comment:
you'd probably want to name this 'grandchild' to avoid confusion with the parent